mysql-test/suite/galera/t/MDEV-34594.test

```diff
@@ -0,0 +1,54 @@
+--source include/galera_cluster.inc
```

Comment:
Would be good to have a comment that briefly describes the test.
Also, having a comment to separate the test phases would improve readability. Something like "Case 1: test with binlog_format ROW", "Case 2: test with binlog_format STATEMENT"